packages/cli-kit/src/public/node/path.ts

```diff
@@ -62,3 +63,20 @@ export function cwd(): string {
   // eslint-disable-next-line rulesdir/no-process-cwd
   return process.env.INIT_CWD ?? process.cwd()
 }
+
+/**
+ * Takes forward slash file path as a input and returns backslash
+ * file path if platform is Windows.
+ *
+ * @param filePath - A file path.
+ * @returns A file path compatible with the current platform architecture.
+ */
+export function platformCompatiblePath(filePath: string): string {
+  const {platform} = platformAndArch()
```

Comment:
is `platformAndArch` an expensive call? because this will be called for every theme file when a theme is being packaged